src/sgame/sg_struct.h

```diff
@@ -452,9 +452,6 @@ struct gentity_t
 
 	bool    pointAgainstWorld; // don't use the bbox for map collisions
 
-	// handle of obstacle that was added to prevent bots to
-	// move toward it. Used for movers and big enough buildings.
-	qhandle_t   obstacleHandle;
```

Comment:
Please comment in your commit message your rationale for removing this.

Reply:
Done